expat/doc/xmlwf.xml

```diff
@@ -423,14 +423,19 @@ supports both.
       </varlistentry>
       <varlistentry>
         <term><option>2</option></term>
-        <listitem><para>An input file was not well-formed or could not be parsed.</para>
+        <listitem><para>One or more input file were not well-formed or could not be parsed.</para>
```

Comment:
s/file/files/

Reply:
Good catch, fixed with the upcoming push. Thanks!